src/main/webapp/app/modeling-assessment-editor/modeling-assessment-conflict/modeling-assessment-conflict.component.ts

```diff
@@ -50,6 +81,90 @@ export class ModelingAssessmentConflictComponent implements OnInit, AfterViewIni
         this.updateSelectedConflict();
     }
 
+    onKeepYours() {
+        this.updateFeedbackInMergedFeedback(
+            this.currentConflict.causingConflictingResult.modelElementId,
+            this.currentConflict.causingConflictingResult.modelElementId,
+            this.currentConflict.causingConflictingResult.result.feedbacks,
+        );
+        this.currentFeedbacksCopy = JSON.parse(JSON.stringify(this.mergedFeedbacks));
+        this.updateCurrentState();
+    }
+
+    onAcceptOther() {
+        this.updateFeedbackInMergedFeedback(
+            this.currentConflict.causingConflictingResult.modelElementId,
+            this.conflictingResult.modelElementId,
+            this.conflictingResult.result.feedbacks,
+        );
+        this.currentFeedbacksCopy = JSON.parse(JSON.stringify(this.mergedFeedbacks));
+        this.updateCurrentState();
+    }
+
+    onFeedbackChanged(feedbacks: Feedback[]) {
+        const elementAssessmentUpdate = feedbacks.find(feedback => feedback.referenceId === this.currentConflict.causingConflictingResult.modelElementId);
+        const originalElementAssessment = this.currentConflict.causingConflictingResult.result.feedbacks.find(
+            feedback => feedback.referenceId === this.currentConflict.causingConflictingResult.modelElementId,
+        );
+        if (elementAssessmentUpdate.credits !== originalElementAssessment.credits) {
```

Comment:
As Array.prototype.find returns undefined if no element is found matching the predicate, originalElementAssessment.credits can cause an "can't read property ... of undefined" error.
Better also check if originalElementAssessment is valid.

Reply:
Same for `elementAssessmentUpdate`.

Reply:
fixed